Add SLAs to public docs

tomskerous opened the pull request:
Added more visible SLAs and signposting for critical incidents to the page, following misleading comments in the press that GaaP don’t have SLAs.

Add in-page links.

P3/P4 Incidents described as “issues”. SLAs identified as for
production tenants only.

Urmi and Jonathan to review.

tomskerous committed:
link from top to critical incident info

- add a name link
- add some copy above fold with internal link

tomskerous committed:
Add table of SLAs

P3/P4 Incidents described as “issues”. SLAs identified as for
production tenants only.

urmishah commented on source/support.html.erb:
```diff
@@ -34,9 +34,12 @@ title: Support
+      </p>
+
+      <p>
+        Tenants are supported differently depending on whether they are in production, and if the request occurs during core hours. For example, only tenants on paid plans who are in production can raise out-of-hours incidents.</p>
```
Suggest changing 'Tenants' to 'Users' or 'Services'

urmishah commented on source/support.html.erb:
```diff
@@ -92,7 +95,84 @@ title: Support
+          <th scope="row">P1<br/>Critical<br/>Incident</th>
+          <td>
+             <ul class="list list-bullet">
+               <li  class="font-xsmall">Apps no longer being served due to an issue with our platform</li>
```
Suggest changing this to 'Applications are unavailable due to a problem with our platform'

urmishah commented on source/support.html.erb:
```diff
@@ -92,7 +95,84 @@ title: Support
+             <ul class="list list-bullet">
+               <li  class="font-xsmall">Apps no longer being served due to an issue with our platform</li>
+               <li class="font-xsmall">Serious security breach on the platform</li>
+               <li class="font-xsmall">You are unable to push an emergency fix to an app due to the PaaS API not being available</li>
```
Suggest changing this to 'You can’t make a critical upgrade to your live production app because the GOV.UK PaaS API isn’t available'

urmishah commented on source/support.html.erb:
```diff
@@ -92,7 +95,84 @@ title: Support
+               <li  class="font-xsmall">Apps no longer being served due to an issue with our platform</li>
+               <li class="font-xsmall">Serious security breach on the platform</li>
+               <li class="font-xsmall">You are unable to push an emergency fix to an app due to the PaaS API not being available</li>
+               <li class="font-xsmall">Your live production app has a P1 issue which cannot be resolved without us</li>
```
Your live production app has a critical P1 issue which cannot be resolved without help from us

urmishah commented on source/support.html.erb:
```diff
@@ -92,7 +95,84 @@ title: Support
+          <td>
+            <ul>
+              <li class="font-xsmall">Can&apos;t update/push apps due to platform issue</li>
+              <li class="font-xsmall">Upstream vulnerabilities</li>
```
Upstream vulnerabilities in GOV.UK PaaS components

urmishah commented on source/support.html.erb:
```diff
@@ -92,7 +95,84 @@ title: Support
+            <ul>
+              <li class="font-xsmall">Can&apos;t update/push apps due to platform issue</li>
+              <li class="font-xsmall">Upstream vulnerabilities</li>
+              <li class="font-xsmall">Elevated error rates</li>
```
Elevated error rates on GOV.UK PaaS platform

urmishah commented on source/support.html.erb:
```diff
@@ -92,7 +95,84 @@ title: Support
+              <li class="font-xsmall">Can&apos;t update/push apps due to platform issue</li>
+              <li class="font-xsmall">Upstream vulnerabilities</li>
+              <li class="font-xsmall">Elevated error rates</li>
+              <li class="font-xsmall">Complete component failure</li>
```
Complete failure of a GOV.UK PaaS component

urmishah commented on source/support.html.erb:
```diff
@@ -92,7 +95,84 @@ title: Support
+              <li class="font-xsmall">Upstream vulnerabilities</li>
+              <li class="font-xsmall">Elevated error rates</li>
+              <li class="font-xsmall">Complete component failure</li>
+              <li class="font-xsmall">Substantial degradation of service</li>
```
Substantial degradation of the GOV.UK PaaS service

urmishah reviewed (changes requested):
Suggesting a few changes:

Tenants are supported differently depending on...
change Tenants  to 'Services' or 'Users'

tomskerous committed:
Respond to Urmi review

- clarify whether it’s paas components failing that trigger a P1, that
it’s elevated error rates on our platform that trigger a P2 etc.

Also added a link to SLAs above the fold and changed the order of they
statuspage/critical incident paragraph as the rhythm felt wrong.

tomskerous committed:
Forgot the #!

Added the # to the a href to make it a working internal link.

jollery committed:
Review of update to support page_paola edited

This pull request has been created after Paola edited the content submitted in a previous pull request by Tom Dolan in this branch. The changes cover:
 - removing content added to the intro section, as this information was duplicated elsewhere on the page
- rewording several of the H2 headers
- general tweaks to some of the content in each section.

The table remains as-is, so it's the same as in the previous pull request.

jollery commented on source/support.html.erb:
```diff
@@ -23,53 +23,40 @@ title: Support
-        live production services. A critical incident is where:
+       Between 5pm and 9am on Monday to Friday, at weekends and during Bank Holidays we provide emergency support for critical incidents affecting live production services.
+       A critical incident is where:
       </p>
```
Some content (bulleted list) isn't displaying here but it's in the code... I don't know how to check whether there's a problem with it or the display.

bleach replied:
This view is only showing you the changes and the bulleted list below this isn't something that's changed.

If you click on the View button above it'll show you the entire file after the changes.

alext committed:
Add govuk_elements table styles.

So that the newly added table on the support page looks right.
